### DISCUSSION TITLE
Can size be learned by objection detection model (YOLOv5)?

### DISCUSSION THREAD

**Discussion by nlm-yuh5** [ultralytics/yolov5#6547] [General]
Hi all,

I have a generic question about object detection. Let's say we are trying to detect two classes of objects that looks exactly the same, the only difference is the size. For example, big air plane and small air plane. (I know this is not a perfect example, as there are features that do differ btw big and small planes. But hopefully you get the gist). How do we get the model to learn the size and therefore to differentiate the two?

**Comment by glenn-jocher** [comment]
@nlm-yuh5 no, object size is generally not a learnable concept unless it is within context, i.e. small tree among large trees.

**Reply by nlm-yuh5** [reply]
Thanks Glenn. That's what I thought, since training label can not include any reference objects.
